install/resource.go

```diff
@@ -388,7 +388,7 @@ func generateDaemonSet(env string) *appsv1.DaemonSet {
 					Containers: []corev1.Container{
 						{
 							Name:  kubearmor,
-							Image: "kubearmor/kubearmor:latest",
+							Image: o.KubearmorImage, // "kubearmor/kubearmor:latest",
```

Comment:
Would it be better to just use the tag from Options?
Something like
```suggestion
							Image: "kubearmor:kubearmor:"+o.KubearmorImage, // "kubearmor/kubearmor:latest",
```
What do you think?

Rest LGTM 👍🏽

Reply:
The use case for me was to use my own image in my own dockerhub repo which is something like `nyrahul/kubearmor:latest` .. WDYT?